### PR TITLE
Update proxy-container-installation-uyuni.adoc

### PR DESCRIPTION
**Pull request by wtmpx**
We haven't mention how to create custom volume.

# Some hints

Consider adding an entry to the `CHANGELOG.md` file in the toplevel directory.
Cosmetic changes such as fixing typos do not need log entries (nevertheless it is important to fix typos, etc.)!
In the `manager-4.3`, the hidden `.changelog` file with a leading dot is still in use.

Add Description, Target branches, and related Links below the following section titles.

In the Description, just enter a summary of why you created this PR (and, if available, add any relevant diagram).

# Description

Short summary of why you created this PR (if you added documentation, please add any relevant diagram).

# Target branches

* Which product version this PR applies to (Uyuni, SUMA 4.3, SUMA MU X.Y.Z, or SUMA development version).  This information can be helpful if `ifeval` statements are needed to publish it for certain products only.
* Does this PR need to be backported? If yes, create an issue for tracking it and add the link to this PR.
* Whenever possible, cross-reference each backport PR here, so that all backports can be easily accessed from the description.

Backport targets (edit as needed):

- master
- 4.3

# Links
- This PR tracks issue #<insert spacewalk issue, if any>
- Related development PR #<insert PR link, if any>

### CODE REVIEW

File: modules/installation-and-upgrade/pages/container-deployment/uyuni/proxy-container-installation-uyuni.adoc [hunk 1]
```diff
@@ -135,6 +135,18 @@ The volumes are named:
...
+
+If /etc/fstab have: 
+/dev/uyuni-proxy/rhn-cache /var/lib/uyuni/proxy-rhn-cache 	xfs 	defaults 0 0
+/dev/uyuni-proxy/squid-cache /var/lib/uyuni/proxy-squid-cache 	xfs 	defaults 0 0
+/dev/uyuni-proxy/tftpboot /var/lib/uyuni/proxy-tftpboot 	xfs 	defaults 0 0
+
+The podman commands to create custom volumes would be: 
+podman volume create --driver local --opt device=/dev/mapper/uyuni--proxy-rhn--cache --opt type=xfs uyuni-proxy-rhn-cache
+podman volume create --driver local --opt device=/dev/mapper/uyuni--proxy-squid--cache --opt type=xfs uyuni-proxy-squid-cache
+podman volume create --driver local --opt device=/dev/mapper/uyuni--proxy-tftpboot --opt type=xfs uyuni-proxy-tftpboot
```
Comment (keichwa) [inline modules/installation-and-upgrade/pages/container-deployment/uyuni/proxy-container-installation-uyuni.adoc:148]:
```suggestion
Run these commands to create custom volumes:

 ---- 
podman volume create --driver local --opt device=/dev/mapper/uyuni--proxy-rhn--cache --opt type=xfs uyuni-proxy-rhn-cache
podman volume create --driver local --opt device=/dev/mapper/uyuni--proxy-squid--cache --opt type=xfs uyuni-proxy-squid-cache
podman volume create --driver local --opt device=/dev/mapper/uyuni--proxy-tftpboot --opt type=xfs uyuni-proxy-tftpboot
----
```

**Review by keichwa — CHANGES_REQUESTED** [review]
Cosmetic suggestions.

File: modules/installation-and-upgrade/pages/container-deployment/uyuni/proxy-container-installation-uyuni.adoc [hunk 2]
```diff
@@ -135,17 +135,29 @@ The volumes are named:
...
+----
+/var/lib/uyuni/proxy-rhn-cache
+/var/lib/uyuni/proxy-squid-cache
+/var/lib/uyuni/proxy-tftpboot 
+----
 
-The podman commands to create custom volumes would be: 
+If [path]``/etc/fstab`` has these entries:
+
+ ----
```
Comment (keichwa) [inline modules/installation-and-upgrade/pages/container-deployment/uyuni/proxy-container-installation-uyuni.adoc:148]:
```suggestion
---- 
```

File: modules/installation-and-upgrade/pages/container-deployment/uyuni/proxy-container-installation-uyuni.adoc [hunk 3]
```diff
@@ -135,6 +135,30 @@ The volumes are named:
...
+
+---- 
+/dev/uyuni-proxy/rhn-cache    /var/lib/uyuni/proxy-rhn-cache    xfs 	defaults 0 0
+/dev/uyuni-proxy/squid-cache  /var/lib/uyuni/proxy-squid-cache  xfs 	defaults 0 0
+/dev/uyuni-proxy/tftpboot     /var/lib/uyuni/proxy-tftpboot     xfs 	defaults 0 0
+----
+
+Run these commands to create custom volumes:
+
+ ----
```
Comment (keichwa) [inline modules/installation-and-upgrade/pages/container-deployment/uyuni/proxy-container-installation-uyuni.adoc:156]:
```suggestion
----
```

File: modules/installation-and-upgrade/pages/container-deployment/uyuni/proxy-container-installation-uyuni.adoc [hunk 4]
```diff
@@ -135,6 +135,30 @@ The volumes are named:
...
+
+----
+/var/lib/uyuni/proxy-rhn-cache
+/var/lib/uyuni/proxy-squid-cache
+/var/lib/uyuni/proxy-tftpboot 
+----
+
+If [path]``/etc/fstab`` has these entries:
+
+----
```
Comment (keichwa) [inline modules/installation-and-upgrade/pages/container-deployment/uyuni/proxy-container-installation-uyuni.adoc:148]:
```suggestion
----
```

**Review by keichwa — CHANGES_REQUESTED** [review]

**Review by keichwa — APPROVED** [review]

File: modules/installation-and-upgrade/pages/container-deployment/uyuni/proxy-container-installation-uyuni.adoc [hunk 5]
```diff
@@ -135,6 +135,30 @@ The volumes are named:
 
 To override default volume settings, create the volumes prior to the first start of the pod using the  [command]``podman volume create`` command.
 
+By default, the [package]``uyuni-proxy-systemd-services`` package creates these three directories that can be used to mount external disks:
```
Comment (mbussolotto) [inline modules/installation-and-upgrade/pages/container-deployment/uyuni/proxy-container-installation-uyuni.adoc:138]: `uyuni-proxy-systemd-services` it's going to be deprecated. Better say something `mgrpxy` for uyuni (instead for 4.3 `uyuni-proxy-systemd-services`  is fine)

**Review by mbussolotto — APPROVED** [review]
just a consideration about change `uyuni-proxy-systemd-services` in favor of `mgrpxy`. The rest seems fine

**Pull request merged**